test/standalone.conf

```diff
@@ -271,10 +271,17 @@ replicationProducerQueueSize=1000
 defaultRetentionTimeInMinutes=0
 
 # Default retention size
-defaultRetentionSizeInMB=0
+defaultRetentionSizeInMB = 0
```

Comment:
We don't use this configuration file to configure the test broker. We always use the environment to change the configurations. So you can remove this change.